bpf/lxc_config.h

```diff
@@ -17,6 +17,9 @@ DEFINE_U32(SECLABEL, 0xfffff);
 DEFINE_U32(SECLABEL_NB, 0xfffff);
 #define SECLABEL_NB fetch_u32(SECLABEL_NB)
 
+DEFINE_U32(POLICY_VERDICT_LOG_FILTER, 0xffff);
+#define POLICY_VERDICT_LOG_FILTER fetch_u32(POLICY_VERDICT_LOG_FILTER)
```

Comment:
Is this needed given its defined in `bpf/lib/policy_log.h` already? If you remove this, the `#ifndef POLICY_VERDICT_LOG_FILTER` guard in `bpf/lib/policy_log.h` can also be removed.

Reply:
In the real case, these configuration are written from control plane, so it will still be in this file, right?

Reply:
Hm, that's a good point. I think that, as long as we move the `DEFINE_U32` in `policy_log.h` before the `#ifdef POLICY_VERDICT_NOTIFY` guard (to ensure it's always present), we should be fine with removing it from `lxc_config.h`. Then, because the symbol will always be in the endpoint's object file, we can also remove the write in `writeStaticData()`. Unless @joestringer prefers that we keep all such `DEFINE_U32`s in the `lxc_config.h` file?

Reply:
If it's going to be (over)written by `writeStaticData()` then I think it should be in lxc_config.h, that way it shows up in `/var/run/cilium/state/EPID/lxc_config.h` and can be retrospectively inspected.

Reply:
I agree that we should ensure that the symbol is unconditionally defined, otherwise you can hit weird upgrade issues when switching the feature on/off.

Reply:
Right, if we remove from `writeStaticData()` it's going to be hard to know the value during debugging :-/
Let's keep it as is then. Sorry for the noise @lzang!

Reply:
Not a problem at all. Thanks a lot for your reviewing!